yoroi.code-workspace

```diff
@@ -58,5 +58,6 @@
     "typescript.preferGoToSourceDefinition": true,
     "typescript.referencesCodeLens.enabled": true,
     "typescript.tsserver.maxTsServerMemory": 12072,
+    "typescript.tsdk": "root/node_modules/typescript/lib",
```

Comment:
is it required? shouldn't it be <root>?

Reply:
generated automatically by Visual Studio Code 🤔 btw it works

Reply:
its to match the repo ts version